vendor/assets/stylesheets/dvl/components/page_header.scss

```diff
@@ -0,0 +1,131 @@
+.page_header {
+  @include cf();
+  border-bottom: 1px solid $lightGray;
+  padding-bottom: 0.6rem;
+  margin-bottom: 1.25rem;
+  position: relative;
+  h2,
+  h3 {
+    margin-bottom: 0;
+    margin-right: 2rem;
+  }
+  h2 {
+    float: left;
+  }
+  h3 {
+    font-weight: 500;
+    color: $darkGray;
+    font-size: 1.3rem;
+    margin-top: 0.3rem;
+  }
+  .secondary_nav,
+  .tertiary_nav {
+    margin-top: 0.5rem;
+  }
+  .secondary_nav {
+    float: left;
+  }
+  .tertiary_nav {
+    float: right;
+  }
+}
+
+// Secondary and tertiary navigation, within page headers.
+
+.secondary_nav,
```

Comment:
If these components are only for usage within `.page_header` (it seems like that's the decision you made), then their class names need to be prefixed with `page_header_`. 

Additionally, you can remove the nested rules on lines 21..30 and put them below this comment. Again, I would refer to that BEM link that I posted earlier until we write some more comprehensive documentation of our CSS methodology.